hlf/chaincode/admission.md

```diff
@@ -0,0 +1,227 @@
+      For detailed informations see [parameter checks](#parameterChecks).
+      (enrollmentId)
+
+
+### Get Admissions for Course
+- ID = getAdmissionsForCourse
+- Send
+    - courseId :: String
+- Receive
+    - admissionList :: List\<[Admission](#Admission)\>
+      - Description: Exhaustive List of all Admissions referencing the course. Empty if none exist.
+
+    - [GenericError](errors.md#GenericError) 
+      ```json
+      {
+        "type": "HLUnprocessableLedgerState",
+        "title": "The state on the ledger does not conform to the specified format"
+      }
+      ```
+      - Description: This error is returned, if the state of data on the ledger is not consistent with the curent model. This error should only occurr if the model changes while the old ledger state remains without modification.
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLUnprocessableEntity",
+        "title": "The following parameters do not conform to the specified format",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+      - Description: This error is returned, if the given parameters are not well formatted, they are listed in invalidParams.  
+      For detailed informations see [parameter checks](#parameterChecks).
+      (courseId)
+
+### Get Admissions for Module
+- ID = getAdmissionsForModule
+- Send
+    - moduleId :: String
+- Receive
+    - admissionList :: List\<[Admission](#Admission)\>
+      - Description: Exhaustive List of all Admissions referencing the module. Empty if none exist.
+
+    - [GenericError](errors.md#GenericError) 
+      ```json
+      {
+        "type": "HLUnprocessableLedgerState",
+        "title": "The state on the ledger does not conform to the specified format"
+      }
+      ```
+      - Description: This error is returned, if the state of data on the ledger is not consistent with the curent model. This error should only occurr if the model changes while the old ledger state remains without modification.
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLUnprocessableEntity",
+        "title": "The following parameters do not conform to the specified format",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+      - Description: This error is returned, if the given parameters are not well formatted, they are listed in invalidParams.  
+      For detailed informations see [parameter checks](#parameterChecks).
+      (moduleId)
+
+## <a id="Models" />Models
+
+### <a id="Admission" />Admission
+```json
+{
+  "admissionId": "0123456:ExampleCourse",
+  "enrollmentId": "0123456",
+  "courseId": "ExampleCourse",
+  "moduleId": "M.1",
+  "timestamp": "2020-09-27_24:59"

```

Comment:
I don't think that 24:59 is a valid time :D

Reply:
```suggestion
  "timestamp": "2020-09-27_23:59"

```